corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -126,19 +126,24 @@ EntrySingleAnswer.prototype.enableReceiver = function (question, options) {
         if (match) {
             var receiveTopic = match[1];
             var receiveTopicField = match[2];
-            question.parentPubSub.subscribe(function (addr) {
-                if (addr === Formplayer.Const.NO_ANSWER) {
+            question.parentPubSub.subscribe(function (message) {
+                if (message === Formplayer.Const.NO_ANSWER) {
                     self.rawAnswer(Formplayer.Const.NO_ANSWER);
-                } else if (addr[receiveTopicField]) {
-                    self.receiveValue(addr[receiveTopicField]);
+                } else if (message, receiveTopicField) {
```

Comment:
Should this be `if (message)`?

Reply:
Actually I'm not sure what I meant there and I'm surprised that was still valid javascript. Good catch!

Reply:
And here I was feeling like a dinosaur and thinking it might be some ES6 magic.

Reply:
Oh I've learned from my ES6 template string minification error to not dabble in the mystic arts. This is purely carelessness haha.